packages/suite-base/src/panels/Plot/PlotCoordinator.ts

```diff
@@ -100,6 +101,10 @@ export class PlotCoordinator extends EventEmitter<PlotCoordinatorEventTypes> {
     this.destroyed = true;
   }
 
+  public setShouldSync({ shouldSync }: { shouldSync: boolean }): void {
```

Comment:
pass only shouldSync in parameters is enough in this case
```suggestion
  public setShouldSync(shouldSync: boolean): void {
```

Reply:
lint rule. We can plan to remove it in the future